src/main/java/io/jawg/osmcontributor/ui/activities/PhotoActivity.java

```diff
@@ -388,43 +399,83 @@ public void onUploadFinishedEVent(final String photoId) {
         flickrPhotoClient.setProperties(oauthRequest.getParams(), new Callback<String>() {
             @Override
             public void success(String s, Response response) {
-                setPhotoTag(photoId);
+                try {
+                    setPhotoTag(photoId);
+                } catch (UnsupportedEncodingException e) {
+                }
             }
 
             @Override
             public void failure(RetrofitError error) {
+                Toast.makeText(PhotoActivity.this, "Erreur lors de l'envoi de la photo, réessayer", Toast.LENGTH_LONG).show();
```

Comment:
You need to use android String resources for i18n.

Add a resource in values/strings/strings.xml with a key/value in both French (in values-fr) and English (default language)
use it in the Toast.makeText API
Toast.makeText(PhotoActivity.this, R.string.picture_sent_failure, Toast.LENGTH_LONG).show();